### ISSUE TITLE
[BUG] Frozen players can unfreeze by spawning a vehicle.

### ISSUE THREAD

**Issue by FireFlame403** [citizenfx/txAdmin#1000]
**txAdmin/FXServer versions:**
tx: v7.3.2

**Describe the bug**
When frozen you can get yourself out of it when you spawn a vehicle.

**Comment by tabarra** [comment]
Hmm I see that this is kinda like a bug in the code, but... does it matter?
As I understand, this means that this only impacts admins, and if they are admins, then does it matter if they can bypass a freeze?

**Comment by FireFlame403** [comment]
Thats fine if it only does that for admins

**Comment by tabarra** [comment]
But, is it the case? Or did I miss something?

**Comment by FireFlame403** [comment]
It lets people drive away when they spawn a car even if they aren't staff

**Comment by yorick2002** [comment]
> It lets people drive away when they spawn a car even if they aren't staff

@FireFlame403 But people that aren't staff should not be able to spawn cars, right?

**Comment by Gravxd** [comment]
> > It lets people drive away when they spawn a car even if they aren't staff
> 
> @FireFlame403 But people that aren't staff should not be able to spawn cars, right?

Lots of servers let normal players spawn vehicles, pretty normal tbh especially with roleplay servers that are menu based

**Comment by tabarra** [comment]
Yeah I can see how that can be an issue... I'll classify this as a bug.  
But I will probably not work on fixing this for the next few months.
  
If anyone is interested in trying to fix it, feel free: https://github.com/tabarra/txAdmin/blob/master/resource/menu/client/cl_freeze.lua

**Comment by Yorick20022** [comment]
there was an attemtpt

https://github.com/tabarra/txAdmin/pull/1003

**Comment by mtgtnt** [comment]
Wouldn't the menu the player is using when spawning a vehicle unfreeze the player? I don't think this would be txadmin issue. The menu that spawns the vehicle should check if player is frozen first and not unfreeze them when doing any action in the menu.
